app/views/campaigns/edit.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="container mt-40 mb-40">
+  <h1 class="text-center">Edit Campaign</h1>
+  <hr class="mb-20">
+  <%= form_for @campaign do |f| %>
+      <div class="form-group">
+        <%= f.label :name %>
+        <%= f.text_field :name, :class => "form-control" %>
```

Comment:
hash rocket syntax (`=>`) was used prior to ruby 1.9, use new syntax instead i.e: `class: "format-control"`